pavement.py

```diff
@@ -13,7 +13,8 @@
 
 def _write_local_settings(lxc_ip, webuiurl):
     local_settings = open(GEM_LOCAL_SETTINGS_TMPL, 'r').read()
-    with open(os.path.join(os.path.expanduser("~"), 'geonode/geonode/'
+    with open(os.path.join(os.path.expanduser("~"), 'oq-platform2/'
+                                                    'openquakeplatform/'
```

Comment:
os.path.join is thought to avoid problem across operative systems, please remove ``/`` from all arguments because it is it's own job.